docs/Model_Assumptions.rst

```diff
@@ -62,6 +62,17 @@ This also means that the costs of the fuel of a diesel generator (input flow) ar
 
 Charge controllers for a :ref:`battery_storage` are defined by two transformers, one for charging and one for discharging.
 
+Heating, Ventilation, and Air Conditioning (HVAC)
+=================================================
+
+Like other conversion assets, devices for heating, ventilation and air conditioning (HVAC) are added as transformers. As the parameters `dispatch_price`, `efficiency` and `installedCap` are assigned to the output flows they need to be given for the nominal heat output of the HVAC.
+
+Different types of HVAC can be modelled. Except for an air source device with ambient temperature as heat reservoir, the device is modelled with two inputs: electricity and heat.
+**NOTE todo: here actually wrong implementation of defining efficiency for the two input busses tba**
```

Comment:
Lets discuss this live - is there already an issue for this?

Reply:
It's explained in the question section in this PR. We can discuss it live and document it in an issue then.